pkg/trace/transform/transform.go

```diff
@@ -102,7 +103,22 @@ func OtelSpanToDDSpanMinimal(
 }
 
 func isDatadogAPMConventionKey(k string) bool {
-	return k == "service.name" || k == "operation.name" || k == "resource.name" || k == "span.type" || k == "deployment.environment.name" || k == semconv.AttributeDeploymentEnvironment
+	return k == "service.name" || k == "operation.name" || k == "resource.name" || k == "span.type" || k == semconv127.AttributeDeploymentEnvironmentName || k == semconv.AttributeDeploymentEnvironment || k == "http.method" || k == "http.status_code"
+}
+
+func mapHTTPAttributes(k string, value string, ddspan *pb.Span) {
+	datadogKey, found := attributes.HTTPMappings[k]
+	switch {
+	case found && value != "":
+		ddspan.Meta[datadogKey] = value
+	case strings.HasPrefix(k, "http.request.header."):
+		key := fmt.Sprintf("http.request.headers.%s", strings.TrimPrefix(k, "http.request.header."))
+		ddspan.Meta[key] = value
+		// Exclude Datadog APM conventions.
+		// These are handled above explicitly.
+	case !isDatadogAPMConventionKey(k):
+		SetMetaOTLP(ddspan, k, value)
```

Comment:
There should be a `default`

or have `mapHTTPAttributes` return a bool to indicate whether to do `SetMetaOTLP` in `OtelSpanToDDSpan`